backend/controller/sql/schema/20240807174508_pubsub_otel_context.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+
+ALTER TABLE topic_events
+    ADD COLUMN otel_context JSONB;
+
+ALTER TABLE async_calls
+    ADD COLUMN otel_context JSONB;
```

Comment:
I wonder if we should keep these generic like `trace_context` or maybe it's better to be `otel` specific in case we have another trace context in the future 🤷